src/main/java/hudson/plugins/brakeman/BrakemanPublisher.java

```diff
@@ -15,7 +15,7 @@
 import java.util.regex.Pattern;
 
 import org.kohsuke.stapler.DataBoundConstructor;
-
```

Comment:
lol yeah, they have a lot of unnecessary new lines and stuff in their files. The general formatting I used was trying to base my format off theirs. I didn't realize theirs was that bad though, I'll re-format my sections to fit the standard, so that it doesn't look like I edited the ENTIRE code base